### ISSUE TITLE
[bug] UnboundLocalError on milestones_content

### ISSUE THREAD

**Issue by deeptibaghel** [webcompat/webcompat.com#2203]
While setting up development error, command "python run.py" throws error as below :
If i initialize milestones_content, it aborts with error "Milestones are not initialized"
It is trying to access milestones folder on my test repo  https://api.github.com/repos/deeptibaghel/webcompat/milestones
which is not there yet.

_Statuses Initialization…
Fetching milestones from Github…
Traceback (most recent call last):
  File "run.py", line 35, in <module>
    from webcompat import app
  File "/Users/deeptibaghel/webcompat.com/webcompat/__init__.py", line 20, in <module>
    app.config.from_object('config')
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/flask/config.py", line 168, in from_object
    obj = import_string(obj)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/werkzeug/utils.py", line 418, in import_string
    __import__(import_name)
  File "/Users/deeptibaghel/webcompat.com/config/__init__.py", line 148, in <module>
    if not initialize_status():
  File "/Users/deeptibaghel/webcompat.com/config/__init__.py", line 60, in initialize_status
    if milestones_content:
UnboundLocalError: local variable 'milestones_content' referenced before assignment_

**Comment by miketaylr** [comment]
Ah, thanks @deeptibaghel. I'd like @karlcow to chime in on how we should maybe fix this when he returns from PTO in a few days. It could either be a documentation bug, or perhaps we can write some code to generate these for a given repo.

As a workaround, you could create the following milestones in your test repo:

https://github.com/webcompat/webcompat-tests/milestones

**Comment by miketaylr** [comment]
(and easier option is to just use the webcmopat/webcompat-tests repo for now)

**Comment by deeptibaghel** [comment]
Thanks @miketaylr .  i created local file milestones.json to make it work, as in my setup requests.get(milestones_url) is also not working, it throws error : 
_It failed with [SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version (_ssl.c:661)!_

Also it is was not getting caught by requests.exceptions.HTTPError, so i changed it to catch all exceptions.
So i am able to run it locally with these changes.

**Comment by miketaylr** [comment]
Hrm, interesting. Looking around... it sounds like you have an older version of openssl? I guess we don't make any requirements for that, and we should.

**Comment by karlcow** [comment]
@deeptibaghel Interesting issue.

## To reproduce (for people with a working configuration):

1. Cut the network
2. Delete data/milestones.json if it already exists.


## What is happening?

1. We try to fetch them from the network and we will write it in a file `milestones.json`
2. If there is an HTTP error, it tries to get it locally from `milestones.json` but indeed if the file is not here, `milestones_content` will never be assigned.

We have a graceful error message already in place but it never reaches it.

```
 UnboundLocalError: local variable 'milestones_content' referenced before assignment
```
is coming from:

https://github.com/webcompat/webcompat.com/blob/61f4ef1be90d7d90eaec1af7a2fb75b1f055aeb9/config/__init__.py#L60


## The Fix

@deeptibaghel Do you feel comfortable enough to create your first pull request? See the instructions in the [Pull Request section of the documentation](https://github.com/webcompat/webcompat.com/blob/master/docs/pr-coding-guidelines.md).

The code to modify is 
https://github.com/webcompat/webcompat.com/blob/61f4ef1be90d7d90eaec1af7a2fb75b1f055aeb9/config/__init__.py#L30-L37

We just need to add after the docstrings
```python
milestones_content = None
```

so the code can return `False`.

## Note

> i created local file milestones.json to make it work

note that if you create it locally and they still do not exist on github it will fail because we need them to manage the issues status. The local milestones.json is a caching mechanism to make it faster, the second time you start the project.

**Comment by karlcow** [comment]
@deeptibaghel 

Could you open a separate issue for the openssl stuff? This is not related to this issue.
In the new issue, please give the results of:

* `python -V`  
* `python -c "import ssl; print(ssl.OPENSSL_VERSION)"`

**Comment by deeptibaghel** [comment]
Thanks @karlcow , have created the PR. My python was indeed using an older version of OPENSSL. After it got fixed, this issue has gone.

**Comment by karlcow** [comment]
Wonderful!.

**Comment by zoepage** [comment]
merged #2221
